### ISSUE TITLE
Hide/show password button partially overlapping long passwords

### ISSUE THREAD

**Issue by isabelrios** [mozilla-lockwise/lockwise-ios#1168]
## Steps to reproduce
Have an entry with a long password
### Expected behavior
The view is shown correctly
### Actual behavior
Please see screenshots:
<img width="150" alt="Screenshot 2019-12-12 at 18 36 15" src="https://user-images.githubusercontent.com/1897507/70735362-65081e80-1d0e-11ea-9d26-97f41dba0684.png">
<img width="150" alt="Screenshot 2019-12-12 at 18 36 28" src="https://user-images.githubusercontent.com/1897507/70735373-6a656900-1d0e-11ea-8873-ecd28ddb1e0d.png">
<img width="150" alt="Screenshot 2019-12-12 at 18 37 32" src="https://user-images.githubusercontent.com/1897507/70735420-8832ce00-1d0e-11ea-8574-36cca30f9578.png">

There is plenty of space on the right side, would not it be nicer if it is moved a little bit? 
cc @changecourse what do you think?

**Comment by changecourse** [comment]
@isabelrios agreed; in the edit view, the hide/show eyeball should anchor to the far right side (where the copy and launch icons are in the view mode.) 

Thanks!
